### PR TITLE
Cv upload

### PR DESCRIPTION
**Pull request by SimonLab**
see #330 

@nelsonic when you have a bit of time do you mind to review this PR? Thanks

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab is this *ready* for review?

**Comment by iteles** [comment]
@nelsonic From the conversations I've had with Simon, it's ready for review.

**Comment by SimonLab** [comment]
Yes ready

File: lib/handlers/cv/post_create.js [hunk 1]
```diff
@@ -1,42 +1,104 @@
...
-    body: {
-      payload: cvObject
-    }
-  }, function (err, response) {
-    // $lab:coverage:off$
-    if(err) {
-        return next(err);
+var extension = request.payload.name.split('.')[1];
+
+var typeMime = {
```
Comment (nelsonic) [inline lib/handlers/cv/post_create.js]: This is fine, but did you look at: https://www.npmjs.com/package/mime-types ?

**Comment by nelsonic** [comment]
Reviewed. looks good. can be simplified. refactor when needed. lets ship.

File: lib/handlers/cv/post_create.js (hunk 1 above)
Comment (SimonLab) [inline lib/handlers/cv/post_create.js]: Thanks for the link, to use it on the next PR

**Pull request merged**